addons/test_mail/tests/test_message_track.py

```diff
@@ -35,6 +35,84 @@ def test_message_track_author(self):
 
         self.assertEqual(self.record.message_ids.author_id, self.partner_admin)
 
+    def test_message_track_default_message(self):
+        def _track_get_message(self, field_names):
```

Comment:
Can't we use `mail.test.track`model and avoid patching `mail.test.ticket` ? Just for simplicity of tests ?